src/lib/__tests__/RoutingManager-test.js

```diff
@@ -1,5 +1,7 @@
+/* globals jsdom */
```

Comment:
what is this used for? looks like your tests are passing without it.

Reply:
`jest-environment-jsdom-global` makes `jsdom` available in our tests ([read more](https://github.com/simon360/jest-environment-jsdom-global#readme)). However, ESLint is not aware of this global variable because this is not part of [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest).